helm/dex-operator/templates/secret.yaml

```diff
@@ -1,15 +1,26 @@
 apiVersion: v1
 stringData:
   credentials: |-
-    - name: mock
+    {{- if .Values.oidc.customer.providers }}
```

Comment:
We allow an array of identity provider credentials for both giantswarm and customers